backend/app/views/spree/admin/shared/_head.html.erb

```diff
@@ -32,7 +32,7 @@
 
 <%= javascript_include_tag 'spree/backend/all' %>
 
-<%= render "spree/admin/shared/translations" %>
+<%= render "spree/admin/shared/js_locale_data" %>
```

Comment:
Good idea 👌🏻 Much more convenient. 

This deserves a CHANGELOG entry though, as people may have modified one of these files. Even deface overrides will not work anymore as the file name changed.